### ISSUE TITLE
Can't use [folder]-[name] on symbolId

### ISSUE THREAD

**Issue by mistic** [kisenka/svg-sprite-loader#108]
Hi @kisenka ,

Congratz for the changes :D I'm trying to setup the version svg-sprite-loader@2.0.1 with for example that config:

```javascript
{
  test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
  include,
  exclude,
  use: {
    loader: 'svg-sprite-loader',
    options: {
      symbolId: '[folder]-[name]',
    }
  },
},
```

However the symbol only get named as '-exampleIcon' because [folder] has no effect. Is this a bug?

**Comment by kisenka** [comment]
Hi @mistic!
If your image is in root build directory - yes, because there is no `[folder]` in this case.

**Comment by mistic** [comment]
Thanks for the answer @kisenka. No, my svg images are always inside a folder. This is my current folder layout:

rootProjectFolder
   -assets
     -svg
       -home
         -warning.svg
         -warning2.svg

So [folder] should return home right? However it isn't...

**Comment by kisenka** [comment]
@mistic it's a bug, just published `2.1.0-2` with fix, please check

**Comment by mistic** [comment]
It's working now 👍
